.changelog/_16677.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+gateway: Fix bug where routes defined in a different namespace than a gateway would fail to register. [[GH-16677](https://github.com/hashicorp/consul/pull/16677)].
```

Comment:
Does this need an enterprise only tag? (I don't actually know if that is a thing in the changelog, just wanted to point it out)

Reply:
👍 We do commonly use a **`(Enterprise only)`** tag in the changelog when appropriate, see https://github.com/hashicorp/consul/blob/main/CHANGELOG.md#1135-december-13-2022 for an example.

Reply:
```suggestion
gateway: **(Enterprise only)** Fix bug where routes defined in a different namespace than a gateway would fail to register. [[GH-16677](https://github.com/hashicorp/consul/pull/16677)].
```